app/models/grda_warehouse/warehouse_reports/time_homeless_for_exit.rb

```diff
@@ -145,13 +145,15 @@ def average_days
     end
 
     private def client_scope
-      apply_filters(client_source.viewable_by(filter.user))
+      apply_filters(client_source.destination_visible_to(filter.user))
```

Comment:
I'd believe we missed this when we updated the viewable_by stuff last.  This exposes a potential issue though, if I'm reading it correctly, because it means you can see all exits for a client you may only be able to see some of the source for.  Does that sound like the results of the change?

Reply:
as discussed, this should be fine because `destination_visible_to` filters by destination and  `merge(GrdaWarehouse::Hud::Project.viewable_by(filter.user))` filters by source